packages/legend-query-builder/src/components/result/tds/QueryBuilderTDSGridResult.tsx

```diff
@@ -278,7 +278,8 @@ export const QueryBuilderTDSGridResult = observer(
       api: DataGridApi<QueryBuilderTDSRowDataType>,
     ): QueryBuilderTDSResultCellData[] => {
       const selectedRanges: DataGridCellRange[] | null = api.getCellRanges();
-      const nodes = api.getRenderedNodes();
```

Comment:
`getRenderedNodes()` only returns the nodes that ag-grid is currently rendering. Due to row virtualization for performance optimization, this function doesn't actually return all the nodes in the grid if there are a large number of nodes. Instead, we need to get all the nodes in the table (not just the rendered ones).

Reply:
Thank you